File: backend/src/main/java/ca/metricalsky/winston/repository/ChannelPropertiesRepository.java
package ca.metricalsky.winston.repository;

import ca.metricalsky.winston.entity.ChannelPropertiesEntity;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface ChannelPropertiesRepository extends JpaRepository<ChannelPropertiesEntity, String> {

}

File: database/src/main/kotlin/ca/metricalsky/winston/database/repository/channel/ChannelPropertiesRepository.kt
package ca.metricalsky.winston.database.repository.channel

import ca.metricalsky.winston.database.entity.channel.ChannelPropertiesEntity
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface ChannelPropertiesRepository: JpaRepository<ChannelPropertiesEntity, String>
